Sample Data/Custom/bloodhoundEnterprise_CL.json

```diff
@@ -0,0 +1,54 @@
+[
+    {
+        "domain_sid": "S-1-5-21-3130019616-2776909439-2417379567",
+        "exposure_index": 1,
+        "tier_zero_count": 1,
+        "critical_risk_count": 1,
+        "id": 17999,
+        "created_at": "2023-05-01T15:35:55.830691Z",
+        "updated_at": "2023-05-01T15:35:55.830691Z",
+        "deleted_at": {
+            "Time": "0001-01-01T00:00:00Z",
+            "Valid": false
+        },
+        "domain_id": "S-1-5-21-3130019616-2776909439-2417379567",
+        "domain_impact_value": 100,
+        "domain_name": "TESTLAB.LOCAL",
+        "domain_type": "active-directory",
+        "exposure": "100",
+        "data_type": "posture"
+    },
+    {
+        "finding_id": "T0FindingId",
+        "domain_id": "S-1-5-21-3130019616-2776909439-2417379567",
+        "path_title": "Example Path Title",
+        "path_type": "Tier Zero Attack Paths",
+        "exposure": 0,
+        "finding_count": 1,
+        "principal_count": 1,
+        "id": 949999,
+        "created_at": "2023-05-01T15:35:43.231504Z",
+        "updated_at": "2023-05-01T15:35:43.231504Z",
+        "deleted_at": {
+            "Time": "0001-01-01T00:00:00Z",
+            "Valid": false
+        },
+        "severity": "Low",
+        "domain_impact_value": 100,
+        "domain_name": "TESTLAB.LOCAL",
+        "domain_type": "active-directory",
+        "data_type": "paths"
+    },
+    {
+        "domain_id": "S-1-5-21-3130019616-2776909439-2417379567",
+        "domain_name": "TESTLAB.LOCAL",
+        "path_id": "T0FindingId",
+        "path_title": "Example Path Title",
+        "group": null,
+        "principal": null,
+        "non_tier_zero_principal": "NON TIER ZERO@TESTLAB.LOCAL",
```

Comment:
Hello @gwhite-so can you change the value of  "non_tier_zero_principal" & "tier_zero_principal".
The ' @ ' is causing validation failure, can you please fix it

Reply:
Hi @v-prasadboke, I'm not sure what the Azure pipeline validation is failing on. The error message is unclear about what the issue is.

Reply:
Hi @v-prasadboke, I removed the '@' sign from the sample data, I'm not sure what portion of the data connector the Azure pipeline is now failing on.